package.json

```diff
@@ -7,13 +7,13 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "start": "concurrently \"npm run start:backend\" \"npm run start:frontend --kubernetes-dashboard:bind_address=$npm_package_config_bind_address\"",
+    "start": "concurrently \"npm run start:backend --kubernetes-dashboard:sidecar_host=$npm_package_config_sidecar_host\" \"npm run start:frontend --kubernetes-dashboard:bind_address=$npm_package_config_bind_address\"",
```

Comment:
Isn't adding `--kubernetes-dashboard:sidecar_host=$npm_package_config_sidecar_host` obsolete if it's already added to the `start:backend` task?

Reply:
Sometimes, I run `start:backend` and `start:frontend` independently. So the new argument is need for both.

Reply:
But `start:backend` already has `--sidecarServerHost $npm_package_config_sidecar_host`. Why is `--kubernetes-dashboard:sidecar_host=$npm_package_config_sidecar_host` needed then for `npm start` if it already executes `npm start:backend`?

My understanding is that wheter we run `npm start` or `npm start:backend` they both should use `sidecarServerHost` parameter defined for `npm start:backend`.

Reply:
I also have thought same as you. But actually `$npm_package_config_sidecar_host` value was not passed to `npm start:backend` when I did not set it from `npm start`. So I concluded we need to set it to both.

Reply:
That's quite confusing. Thank you for checking that.